jscomp/others/js_option.ml

```diff
@@ -59,5 +59,27 @@ let andThen f x =
   | None -> None 
   | Some x -> f x [@bs]
 
+let map f x =
+  match x with
+  | None -> None
+  | Some x -> Some (f x [@bs])
```

Comment:
`andThen`  is used by Elm and Rust, at the very least. And `bind` is rightly discarded by any new language seeking some actual adoption. I'd be fine with `flatMap` (and snake_case too) but `andThen` (and camelCase) is more intuitive to a lot more developers.